svgnative/include/svgnative/SVGRenderer.h

```diff
@@ -98,26 +98,65 @@ struct Gradient;
 class Transform;
 class Path;
 class Shape;
+class Interval;
```

Comment:
Can be moved to the new Rect.h header

Reply:
a new file rect.h and rect. cpp is made and moved the relevant contents.